servers/rendering/renderer_scene_cull.cpp

```diff
@@ -637,20 +637,22 @@ void RendererSceneCull::instance_set_base(RID p_instance, RID p_base) {
 				instance->base_data = geom;
 				geom->geometry_instance = scene_render->geometry_instance_create(p_base);
 
-				geom->geometry_instance->set_skeleton(instance->skeleton);
-				geom->geometry_instance->set_material_override(instance->material_override);
-				geom->geometry_instance->set_material_overlay(instance->material_overlay);
-				geom->geometry_instance->set_surface_materials(instance->materials);
-				geom->geometry_instance->set_transform(instance->transform, instance->aabb, instance->transformed_aabb);
-				geom->geometry_instance->set_layer_mask(instance->layer_mask);
-				geom->geometry_instance->set_lod_bias(instance->lod_bias);
-				geom->geometry_instance->set_transparency(instance->transparency);
-				geom->geometry_instance->set_use_baked_light(instance->baked_light);
-				geom->geometry_instance->set_use_dynamic_gi(instance->dynamic_gi);
-				geom->geometry_instance->set_use_lightmap(RID(), instance->lightmap_uv_scale, instance->lightmap_slice_index);
-				geom->geometry_instance->set_cast_double_sided_shadows(instance->cast_shadows == RS::SHADOW_CASTING_SETTING_DOUBLE_SIDED);
-				if (instance->lightmap_sh.size() == 9) {
-					geom->geometry_instance->set_lightmap_capture(instance->lightmap_sh.ptr());
+				if (geom->geometry_instance) {
```

Comment:
Why is this change needed? As far as I can tell a geometry_instance is allocated for every InstanceGeometryData immediately after creation. Is there a possibility that geometry_instance gets freed while the InstanceGeometryData does not?

Reply:
Oops, I see you have already flagged this line

Reply:
Because the dummy renderer doesn't instance it because it doesn't implement it, it returns a NULLPTR

I'm changing that right now..

Reply:
Ok added proper geometry instance support as well.